changebot/blueprints/tests/test_changelog_helpers.py

```diff
@@ -73,7 +73,9 @@ def test_review_long_changelog():
     assert len(issues) == 0
 
 
-@pytest.mark.parametrize('label', ['no-changelog-entry-needed', 'Affects-dev'])
+@pytest.mark.parametrize('label', ['no-changelog-entry-needed',
+                                   'Affects-dev',
```

Comment:
Ohh, stupid me, actually we could easily just use the "Affect-dev" label...

Reply:
Perhaps, but it is still non-intuitive. Regardless, I don't mind closing this PR if you want to just use "Affects-dev" (though it should be documented in the release procedures?). Please let me know.

Reply:
no, this is definitely clearer, it was just my heureka moment. 

So it's up to your or @astrofrog's preference, I'm happy either way.

Reply:
Actually @bsipocz , "affects dev" won't fix this. I thought of the logic again and the check for "affects dev" happens last in the if "tree", so that is a no-go.